qiskit/providers/fake_provider/fake_backend.py

```diff
@@ -553,11 +553,11 @@ def run(self, run_input, **kwargs):
             )
         if pulse_job:
             if _optionals.HAS_AER:
-                from qiskit.providers import aer
-                from qiskit.providers.aer.pulse import PulseSystemModel
+                from qiskit_aer import Aer
+                from qiskit_aer.pulse import PulseSystemModel
```

Comment:
I don't think we have any tests that run pulse jobs on fake backends, and I think that's why this line doesn't fail, but `qiskit_aer.pulse` was deprecated in qiskit-aer 0.12 and removed without replacement in qiskit-aer 0.13 (well, the replacement is to use qiskit dynamics). I cannot add an inline suggestion for the unchanged parts but I'd do the same as in line 350:

```
        if pulse_job: 
            raise QiskitError("Pulse simulation is currently not supported for V1 fake backends.")
```

Reply:
Oh, I didn't know that, thanks!  I've replaced the messages in both this and the V2 backends to be generally "pulse simulation not supported", and put in a release note about that.  I'll update #11439 accordingly as well.